objects/obj_mass_equip/Step_0.gml

```diff
@@ -1,15 +1,17 @@
 
 if (obj_controller.settings=0) or (obj_controller.menu!=23) then instance_destroy();
 
+var romanNumerals;
+romanNumerals= scr_roman_numerals();
+
 if (engage=true){
-    var co,i;co=-1;i=0;
-    
-    repeat(11){co+=1;i=0;
+    for(var co=0, i=0; co<11; co++){
         if (role_number[co]>0){
-            repeat(300){i+=1;
+			for(i=0; i<300; i++){
```

Comment:
also, this should start from 1. Previously, the first pass through the `repeat(300){...` incremented `i` at the start, so it was always at least 1 in the loop